pkg/test/environment.go

```diff
@@ -191,6 +191,7 @@ func (env *Environment) Reset() {
 	env.PricingAPI.Reset()
 	env.PricingProvider.Reset()
 	env.ImageProvider.Reset()
+	env.MockSkuClientSignalton.SKUClient.Reset()
```

Comment:
```suggestion
	env.MockSkuClientSignleton.SKUClient.Reset()
```
Looks like an old type (would need fixing in the struct as well)

Reply:
Fixed the struct as well.